src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userSer = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userSer.findByEmail(email);
+        if (user != null && password.equals(user.getPassword())) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
There is no need in if/else when we need to return boolean value
```suggestion
        return user != null && password.equals(user.getPassword());
```